### PR TITLE
Added some content to mech dept. and edited discord link

### PR DESCRIPTION
**Pull request by Arjun-G-04**
## 📑 Description

- Added some content to mech dept sem 1
- Added basic home page for mech dept
- Fixed typo in the blog regarding discord link

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

## ℹ Additional Information

If the discord server is ready, kindly add the invite link to the blog!

### CODE REVIEW

File: blog/Welcome.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Never made an open source contribution before? Wondering how contributions work
 
 ### Where can I go for help 🤨?
 
-If you need help, you can ask questions on our [discord](https://dicord.com) channel.
+If you need help, you can ask questions on our [discord](https://discord.com) channel.
```
Comment (jaiakash) [inline blog/Welcome.md]: We have to add whatsapp link to it.

**Review by jaiakash — APPROVED** [review]

**Comment by jaiakash** [comment]
Hey @Arjun-G-04, Thanks for your contribution. Just add the WhatsApp community link then we are good to go to merge your changes.

File: blog/Welcome.md (hunk 1 above)
Comment (jaiakash) [reply blog/Welcome.md]: Instead of Discord, Add Whatsapp [link](https://chat.whatsapp.com/CQ0LXqibJjCESfdnWrIbRR) to it.

Comment (Arjun-G-04) [reply blog/Welcome.md]: okay!

**Comment by Arjun-G-04** [comment]
Done!

**Pull request merged**